libs/core/langchain_core/chat_history.py

```diff
@@ -102,6 +101,8 @@ async def aget_messages(self) -> List[BaseMessage]:
         In general, fetching messages may involve IO to the underlying
         persistence layer.
         """
+        from langchain_core.runnables.config import run_in_executor
```

Comment:
Should be usable in global namespace I think since it's using runnables.config rather than runnables?